wp-post-meta-revisions.php

```diff
@@ -59,7 +59,7 @@ public function _wp_autosave_post_revisioned_meta_fields( $new_autosave ) {
 		 * The post data arrives as either $_POST['data']['wp_autosave'] or the $_POST
 		 * itself. This sets $posted_data to the correct variable.
 		 */
-		$posted_data = isset( $_POST['data'] ) ? $_POST['data']['wp_autosave'] : $_POST; // WPCS: CSRF ok. input var ok. sanitization ok.
+		$posted_data = isset( $_POST['data']['wp_autosave'] ) ? $_POST['data']['wp_autosave'] : $_POST; // WPCS: CSRF ok. input var ok. sanitization ok.
```

Comment:
will this throw an error if `$_POST['data']` is not set? maybe isset( $_POST['data'] ) && isset( $_POST['data']['wp_autosave'] )

Reply:
No, that's the magic of `isset()` :smiley: 

https://3v4l.org/JS80T

Reply:
Ah, nice! I guess my JavaScript habits have me doing_it_wrong in php. Thanks for illuminating me.

Reply:
We all learn everyday.

And a little trick in case you do need to check for the existence of various variables: `isset()`, like it's sibling `unset()`, can take any number of arguments.

So, instead of writing this:
```php
if ( isset( $a ) && isset( $b ) && isset( $c ) && $a > $b && b > $c ) {}
```

... you can use:
```php
if ( isset( $a, $b, $c ) && $a > $b && b > $c ) {}
```